ai_edge_torch/generative/layers/feed_forward.py

```diff
@@ -93,3 +93,38 @@ def forward(self, x):
       torch.Tensor: output tensor after feedforward.
     """
     return self.w2(self.act(self.w1(x)) * self.w3(x))
+
+
+class SharedGatedFeedForward(nn.Module):
+  """Shared Gated Feedforward with customizable activation."""
+
```

Comment:
is there a paper/github reference for this feedforward function we can put here?

Reply:
It turns out the implementation was wrong.
`QUICK_GELU` should be `QUICK_GELU(x) = x * F.sigmoid(1.702 * x)`, so it falls into `SEQUENTIAL` type.

Removed `SharedGatedFeedForward`.

Reply:
can you remove this class then?

Reply:
removed